### PR TITLE
# Make `body: Option[String]`

### PR DESCRIPTION
**Pull request by lloydmeta**
You can create a PR with `"body": null` if you use the API, or use a tool
that uses the API.

This PR makes `body: Option[String]`

```shell
curl -X GET https://api.github.com/repos/lloydmeta/gh-test-repo/pulls
[
  {
    "url": "https://api.github.com/repos/lloydmeta/gh-test-repo/pulls/3",
    // snip
    "body": null,
    // snip
  },
  {
    "url": "https://api.github.com/repos/lloydmeta/gh-test-repo/pulls/1",
    // snip
    "body": "",
    // snip
]
```

### CODE REVIEW

**Comment by BenFradet** [comment]
@lloydmeta do you need a release?

File: github4s/shared/src/test/scala/github4s/integration/GHPullRequestsSpec.scala [hunk 1]
```diff
@@ -35,6 +35,18 @@ trait GHPullRequestsSpec[T] extends BaseIntegrationSpec[T] {
     })
   }
 
+  "PullRequests >> List" should "return a right response when a valid repo is provided but not all pull requests have body" in {
+    val response =
+      Github(accessToken).pullRequests
+        .list("lloydmeta", "gh-test-repo", List(PRFilterOpen))
```
[the remark this answers is not in the document]
Comment (lloydmeta) [reply github4s/shared/src/test/scala/github4s/integration/GHPullRequestsSpec.scala]: LOL, yeah, I made that repo just so there would be zero chance of it changing :)

**Comment by lloydmeta** [comment]
@BenFradet Yea, a release would be much appreciated. 

Just for some context, I'm trying to build a tool that would allow me to run Scalafmt on files that aren't currently involved in any recent PRs and submit a PR for formatting. The idea being to unintrusively format a large codebase.  I'm filing PRs to github4s as I make progress.

**Comment by BenFradet** [comment]
interesting stuff, we'll release 0.18.3 :+1:

**Pull request merged**